pages.tr/common/7z.md

```diff
@@ -0,0 +1,36 @@
+# 7z
+
+> Yüksek sıkıştırma oranına sahip dosya sıkıştırıcısı.
+> Daha fazla bilgi: <https://www.7-zip.org/>.
+
+- Dosya veya dizin arşivle:
+
+`7z a {{archived.7z}} {{yoldan/dosya_veya_dizine}}`
```

Comment:
Please translate `archived`, I know it's not a good filename. You can use `compressed_file` for the translation, or something similar.